code/datums/uplink_item.dm

```diff
@@ -991,6 +991,7 @@ var/list/uplink_items = list()
 	reference = "BRHS"
 	item = /obj/item/weapon/storage/box/syndie_kit/hardsuit
 	cost = 8
+	excludefrom = list(/datum/game_mode/nuclear)
```

Comment:
They can accomplish far worse by purchasing a syndicate balloon.

We don't ward against all forms of ignorance and stupidity =p